files/.htaccess

```diff
@@ -1,6 +1,5 @@
 # Turn off all options we don't need.
-Options None
-Options +SymLinksIfOwnerMatch
+Options -Indexes -ExecCGI -Includes
 
```

Comment:
If you're following D8/9, why do you not have -MultiViews in there?

Reply:
@oadaeh - I'm not following D8/9 - I'm following what's in /core/includes/file.inc lines 539-555
https://github.com/backdrop/backdrop/blob/c144627cadb416d979aaa89bc9659c0cb04a7b2b/core/includes/file.inc#L539-L555
They need to match if security review test can be meaningful.